lib/block_view.go

```diff
@@ -2128,6 +2106,36 @@ func (bav *UtxoView) _connectBasicTransferWithExtraSpend(
 	return totalInput, totalOutput, utxoOpsForTxn, nil
 }
 
+func (bav *UtxoView) _helpVerifySignature(txn *MsgDeSoTxn, blockHeight uint32) error {
```

Comment:
`_helpVerifySignature` is a bit ambiguous. The word help implies that this function performs a subset of signature verifications, but not all. Is that correct?

I suggest renaming it to something like `_verifyTxnSignature` for clarity
```suggestion
func (bav *UtxoView) _verifyTxnSignature(txn *MsgDeSoTxn, blockHeight uint32) error {
```

Reply:
Hm, yeah we could change the name. But I'm not sure that `_verifyTxnSignature` is better. There's the primary signature verification utility called `_verifySignature`, and we wouldn't want to confuse the two together.